anagram/tests/anagram.rs

```diff
@@ -1,11 +1,6 @@
-#![crate_name = "anagram_test"]
-#![crate_type = "lib"]
-#![feature(collections)]
-
-mod anagram;
+extern crate anagram;
 
 #[test]
-#[ignore]
```

Comment:
It's a bit of a matter of language style though. Table driven testing is fairly embraced in Go's culture but it doesn't really chime with styles like BDD (which saddens me, since table driven is hugely effective test design). BDD is what the javascript track uses since BDD frameworks like Jasmine are very popular there.